telethon_examples/interactive_telegram_client.py

```diff
@@ -270,6 +271,23 @@ def run(self):
                     else:
                         print('No profile picture found for this user!')
 
+                elif msg == '!i':
+                    blacklist_attrs = ['from_reader', 'on_response', 'pretty_format', 'resolve', 'rpc_error',
```

Comment:
This is horribly ugly. You should use `entity.to_dict()` and iterate over that, no blacklist needed.

Reply:
Well then how do we know which attributes are interessting and which ones to ignore? __Some__ list of attribs must be in there, in any form

Reply:
Okay sure but using `.to_dict()` would save you from knowing library internals, like `from_reader`, `on_response`, `pretty_format`, `resolve` and `rpc_error`. Oh, look, all of the relevant ones.